tools/data_manipulation/read_netcdf/read_netcdf/netcdf_read.xml

```diff
@@ -0,0 +1,259 @@
+                </when>
+                <when value="be">
+                    <param name="t1" type="float" value="0" label="Inferior threshold"/>
+                    <param name="t2" type="float" value="0" label="Superior threshold"/>
+                </when>
+                <when value="e">
+                    <param name="value" type="float" value="0" label="Value"/>
+                </when>
+                <when value="g">
+                    <param name="value" type="float" value="0" label="Value"/>
+                </when>
+                <when value="ge">
+                    <param name="value" type="float" value="0" label="Value"/>
+                </when>
+                <when value="l">
+                    <param name="value" type="float" value="0" label="Value"/>
+                </when>
+                <when value="le">
+                    <param name="value" type="float" value="0" label="Value"/>
+                </when>
+            </conditional>
+        </repeat>
+
+    </inputs>        
+
+    <outputs>
+        <collection type="list" name="output">
+            <discover_datasets pattern="__designation_and_ext__" visible="false" format="tabular" directory="output_dir"/>
+            <filter>condi_source_coord['coord_source'] == 'coord_from_file'</filter>
+        </collection>
+        <data name="simpleoutput" from_work_dir="final.tabular" format="tabular">
+            <filter>condi_source_coord['coord_source'] == 'coord_from_stdin'</filter>
+        </data>
+    </outputs>
+
+
+    <tests>
+        <test>
+             <param name="input" value="dataset-ibi-reanalysis-bio-005-003-monthly-regulargrid_1510914389133.nc"/>
+             <param name="var_tab" value="var_tab_dataset-ibi"/>
+             <param name="var" value="phy"/>
+             <param name="dim_tab" value="tab_dim_phy_dataset-ibi"/>
+             <param name="coord" value="yes_cut_coord"/>
+             <param name="lat_dim" value="latitude"/>
+             <param name="lat_val" value="44.0"/>
+             <param name="lon_dim" value="longitude"/>
+             <param name="lon_val" value="-2.0"/>
+             <param name="output" value="Test1.tabular"/>
+        </test>
+        <test>
+             <param name="input" value="dataset-ibi-reanalysis-bio-005-003-monthly-regulargrid_1510914389133.nc"/>
+             <param name="var_tab" value="var_tab_dataset-ibi"/>
+             <param name="var" value="nh4"/>
+             <param name="dim_tab" value="tab_dim_nh4_dataset-ibi"/>
+             <param name="coord" value="nope"/>
+             <param name="dim" value="time"/>
+             <param name="comparator" value="e"/>
+             <param name="value" value="7272.0"/>
+             <param name="dim" value="latitude"/>
+             <param name="comparator" value="ge"/>
+             <param name="value" value="45.0"/>
+             <param name="output" value="Test2.tabular"/>
+        </test>
+
```

Comment:
can you clean this empty spaces here up?

Reply:
done